SECURITY.rst

```diff
@@ -0,0 +1,124 @@
+Security
+________
+
+This document contains information on how to report security vulnerabilities in Astronomer Cosmos and
+how security issues reported to the Astronomer security team are handled.
+If you would like to learn more, please refer to `https://www.astronomer.io/security/ <https://www.astronomer.io/security/>`_.
+
+At Astronomer, we recognize the critical nature of security and view it as a transparent and collaborative effort.
+If you have any concern around the security of any Astronomer public repository or believe you have uncovered a vulnerability,
+please email `oss_security@astronomer.io <mailto:oss_security@astronomer.io>`_.
+
+.. warning::
+    Note: Due to the nature of some security vulnerabilities, we ask that you not create a GitHub issue.
+
+
+Use of Email for Vulnerability Disclosure
+=========================================
+
+Only use the above email to disclose security vulnerabilities.
+We do not accept bug reports, security implementation questions or other security-related issues at this email address.
+If you are a customer of Astronomer, please reach out to your account team if you have any security-related questions or
+issues other than vulnerabilities, and they can assist you. Otherwise, this codebase is provided ‘as-is’ in accordance
+with its licensing structure.
+
+Scope
+=====
+
+When submitting vulnerabilities, please ensure that it is within scope. Out of scope will be ignored.
```

Comment:
Within scope of what? The project? The tool? I think this might need one more word.

Reply:
In and out of scope based on the definitions below.